### PR TITLE
update sync.py to support aws_s3_endpoint

### PR DESCRIPTION
**Pull request by ryansteakley**
**Which issue is resolved by this Pull Request:**
Resolves #

**Description of your changes:**
Tested with govcloud using static credentials, since govclouds default s3_endpoint is not s3.amazonaws.com would not work unless you can specify the actual aws_s3_endpoint value that gov-cloud uses.

**Testing:**
- [ ] Unit tests pass
- [ ] e2e tests pass
- Details about new tests (If this PR adds a new feature)
- Details about any manual tests performed

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: awsconfigs/apps/pipeline/s3/sync.py [hunk 1]
```diff
@@ -345,6 +347,10 @@ def sync(self, parent, children):
                                             "name": "AWS_REGION",
                                             "value": f"{minio_service_region}"
                                         },
+                                        {
```
Comment (surajkota) [inline awsconfigs/apps/pipeline/s3/sync.py:350]: shouldnt the export for AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY be removed now?

Comment (ryansteakley) [reply awsconfigs/apps/pipeline/s3/sync.py:350]: why is that? I believe meghna had added those, but we are still maintaining the existing static credentials approach in this release.

Comment (surajkota) [reply awsconfigs/apps/pipeline/s3/sync.py:350]: We should separate this or make this conditional

**Review by surajkota — APPROVED** [review]
lg for now

**Pull request merged**